SphereEversion/ToMathlib/Analysis/InnerProductSpace/Projection.lean

```diff
@@ -111,11 +111,8 @@ variable {x₀ x : E}
 theorem mem_orthogonal_span_singleton_iff {x₀ x : E} : x ∈ {.x₀}ᗮ ↔ ⟪x₀, x⟫ = 0 := by
   simp only [mem_orthogonal, forall_mem_span_singleton, inner_smul_left, RCLike.conj_to_real,
     mul_eq_zero]
-  constructor
-  · intro h
-    simpa using h 1
-  · intro h t
-    exact Or.inr h
+  refine ⟨fun h ↦ ?_, fun h t ↦ Or.inr h⟩
+  simpa using h 1
```

Comment:
You could inline this into the previous line, if you want.

Reply:
Done.